x-pack/platform/plugins/shared/index_management/public/application/services/api.ts

```diff
@@ -482,3 +488,21 @@ export function useLoadInferenceEndpoints() {
     method: 'get',
   });
 }
+
+export const startReindex = (sourceIndexName: string, lookupIndexName: string) => {
+  return reindexService.startReindex({
+    indexName: sourceIndexName,
+    newIndexName: lookupIndexName,
```

Comment:
I think it would be a good idea to delete the old index and provide an alias from old to new. This is available via one of the startReindex params